contracts/staking/contracts/src/vaults/ZrxVault.sol

```diff
@@ -70,7 +70,7 @@ contract ZrxVault is
     }
 
     /// @dev Sets the Zrx proxy.
-    /// Note that only the contract owner can call this.
+    /// Note that only the contract staker can call this.
```

Comment:
Nit: This should probably still be `owner`. With this in mind, should this be changed to `onlyAuthorized`?